src/ensembl/src/content/app/browser/browser-cog/BrowserCogList.tsx

```diff
@@ -57,29 +58,29 @@ type BrowserCogListProps = {
 
 export const BrowserCogList = (props: BrowserCogListProps) => {
   const { browserCogTrackList } = props;
-  const listenBpaneScroll = (payload: BrowserScrollPayload) => {
-    if (
-      payload.action === BrowserToChromeMessagingAction.UPDATE_SCROLL_POSITION
-    ) {
-      props.updateCogList(payload.delta_y);
-    }
-    if (
-      payload.action === BrowserToChromeMessagingAction.UPDATE_TRACK_POSITION
-    ) {
-      props.updateCogTrackList(payload.track_y);
-    }
+
+  const updateCogsPosition = (payload: CogScrollPayload) => {
+    props.updateCogList(payload.delta_y);
+  };
+
+  const updateCogTracksPosition = (payload: CogTrackScrollPayload) => {
```

Comment:
Do we need to use `Cog` here?

Reply:
Don't know. For whatever reason, the action that is triggered by this message is called `updateCogTrackList` 🤷‍♂️ 

I am equally unsure whether we want the word `Cog` in `CogScrollPayload` type. We are using it to move the cogs; but perhaps the better name for it would be `VerticalScrollPayload`. We need to find out exactly what these messages do, and what was the reason for using the word `cog` in the name of the `updateCogTrackList` action.

Reply:
Cool, I see if I can get this changed in the branch I'm working on.